packages/codegen/src/generateEntityCodegenFile.ts

```diff
@@ -407,9 +407,11 @@ export function generateEntityCodegenFile(config: Config, meta: EntityDbMetadata
       }
 
       populate<H extends ${LoadHint}<${entityName}>>(hint: H): Promise<${Loaded}<${entityName}, H>>;
+      populate<H extends ${LoadHint}<${entityName}>>(opts: { hint: H, forceReload?: boolean }): Promise<${Loaded}<${entityName}, H>>;
```

Comment:
Adding overloads so that `CustomCollection`s can pass in `forceReload` opt if they need/want